src/libs/RequestThrottle.ts

```diff
@@ -1,28 +1,22 @@
-import _ from 'underscore';
 import CONST from '../CONST';
+import {generateRandomInt} from './NumberUtils';
 
 let requestWaitTime = 0;
 
-function clear() {
+function clear(): void {
```

Comment:
Do we really need type this one line function?

<img width="275" alt="Screenshot 2023-09-11 at 15 47 24" src="https://github.com/Expensify/App/assets/57314631/cee46711-92b2-4da9-9c1b-15589613cd50">

Reply:
I agree, we should remove this type.

Reply:
Ya, that makes sense, just did it